opendm/config.py

```diff
@@ -285,32 +304,61 @@ def config():
 
     parser.add_argument('--use-exif',
                         action='store_true',
-                        default=False,
+                        default=defaultSettings['georeferencing']['use_exif'],
                         help=('Use this tag if you have a gcp_list.txt but '
                               'want to use the exif geotags instead'))
 
     parser.add_argument('--orthophoto-resolution',
                         metavar='<float > 0.0>',
-                        default=20.0,
+                        default=defaultSettings['orthophoto']['resolution'],
                         type=float,
                         help=('Orthophoto ground resolution in pixels/meter'
                               'Default: %(default)s'))
 
-    parser.add_argument('--zip-results',
+    parser.add_argument('--orthophoto-target-srs',
```

Comment:
Unfortunately target_srs doesn't work because gdal_translate doesn't have that option. @smathermather if you have any suggestions?